### PR TITLE
Improve unsave version mark on archive view

### PR DESCRIPTION
**Pull request by ivy-lli**
Before:
image

After:
image

### CODE REVIEW

**Comment by ivy-lli** [comment]
Changed it again, so it also support to show the real issue with a link to more info (provided by the unsafe.version file):

Empty unsafe.version file (e.g 6.0):
image

unsafe.version file with issues (e.g 8.0.1):
image

File: src/app/domain/ReleaseInfo.php [hunk 1]
```diff
@@ -54,6 +54,14 @@ public function isUnsafeVersion(): bool
   {
     return file_exists($this->getPath() . '/unsafe.version');
   }
+  
+  public function getUnsafeReasons(): array
+  {
+    $unsaveFile = $this->getPath() . '/unsafe.version';
```
Comment (alexsuter) [inline src/app/domain/ReleaseInfo.php:60]: line 55 and line 60 has some duplicated code. can you move it to an own method?

private function getUnsafeVersionPath(): string {
...
}

File: src/web/releases/ivy/8.0.1/unsafe.version [hunk 2]
```diff
@@ -0,0 +1,10 @@
+[
```
Comment (alexsuter) [inline src/web/releases/ivy/8.0.1/unsafe.version:1]: or maybe we introduce a new file for this?
- security-issue.json
- security.json
- ... ?

**Review by alexsuter — APPROVED** [review]

File: src/web/releases/ivy/8.0.1/unsafe.version (hunk 2 above)
Comment (ivy-lli) [reply src/web/releases/ivy/8.0.1/unsafe.version:1]: would you prefer a complete new file (replacement for the unsafe.version) or an additional one?

Comment (alexsuter) [reply src/web/releases/ivy/8.0.1/unsafe.version:1]: best would be to replace the existing. convert it directly on the server.
but maybe it is not worth.
its ok to me to merge it now.

**Pull request merged**